// react-block/lib/blocks/hero.js
/**
 * Hero Block
 * @class
 * @author Andrew Wilson 
 */

'use strict';

Object.defineProperty(exports, '__esModule', {
	value: true
});

var _createClass = (function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ('value' in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; })();

var _get = function get(_x, _x2, _x3) { var _again = true; _function: while (_again) { var object = _x, property = _x2, receiver = _x3; desc = parent = getter = undefined; _again = false; if (object === null) object = Function.prototype; var desc = Object.getOwnPropertyDescriptor(object, property); if (desc === undefined) { var parent = Object.getPrototypeOf(object); if (parent === null) { return undefined; } else { _x = parent; _x2 = property; _x3 = receiver; _again = true; continue _function; } } else if ('value' in desc) { return desc.value; } else { var getter = desc.get; if (getter === undefined) { return undefined; } return getter.call(receiver); } } };

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError('Cannot call a class as a function'); } }

function _inherits(subClass, superClass) { if (typeof superClass !== 'function' && superClass !== null) { throw new TypeError('Super expression must either be null or a function, not ' + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var Hero = (function (_Component) {
	_inherits(Hero, _Component);

	_createClass(Hero, null, [{
		key: 'propRef',
		value: {
			title: 'HTML',
			subtitle: 'HTML',
			buttonText: '',
			buttonLink: ''
		},

		/** @constructor */
		enumerable: true
	}]);

	function Hero(props) {
		_classCallCheck(this, Hero);

		_get(Object.getPrototypeOf(Hero.prototype), 'constructor', this).call(this, props);
	}

	// export default class HeaderBlock {
	// render() {
	// return <Header />
	// }
	// }

	/**
  * Render for Header
  * @returns {Function}
  */

	/*
 YouTube video
 		<div style={{position: 'fixed', zIndex: -99, width: '100%', height: '100%'}}>
 	  <iframe frameborder="0" height="100%" width="100%" 
 	    src="https://youtube.com/embed/aLQeqyOK4lE?autoplay=1&controls=0&showinfo=0&autohide=1">
 	  </iframe>
 	</div>
 */

	_createClass(Hero, [{
		key: 'render',
		value: function render() {
			return _react2['default'].createElement(
				'div',
				{ className: 'bg-primary dk', style: this.props.styles },
				_react2['default'].createElement(
					'div',
					{ className: 'text-center wrapper' },
					_react2['default'].createElement(
						'div',
						{ className: 'm-t-xl m-b-xl' },
						_react2['default'].createElement(
							'div',
							{ className: 'text-uc h1 font-bold inline' },
							_react2['default'].createElement(
								'div',
								{ className: 'pull-left m-r-sm text-white' },
								this.props.title
							)
						),
						_react2['default'].createElement(
							'div',
							{ className: 'h4 text-muted m-t-sm' },
							this.props.subtitle
						)
					),
					_react2['default'].createElement(
						'p',
						{ className: 'text-center m-b-xl' },
						_react2['default'].createElement(
							'a',
							{ href: this.props.buttonLink, className: 'btn btn-lg btn-warning b-white bg-empty m-sm' },
							this.props.buttonText
						)
					)
				),
				_react2['default'].createElement(
					'div',
					{ className: 'padder' },
					_react2['default'].createElement('div', { className: 'hbox' })
				)
			);
		}
	}]);

	return Hero;
})(_react.Component);

exports['default'] = Hero;
module.exports = exports['default'];
