// react-block/lib/BlockStore.js
'use strict';

Object.defineProperty(exports, '__esModule', {
	value: true
});

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _blocksHeader = require('./blocks/header');

var _blocksHeader2 = _interopRequireDefault(_blocksHeader);

var _blocksParagraph = require('./blocks/paragraph');

var _blocksParagraph2 = _interopRequireDefault(_blocksParagraph);

var _blocksImage = require('./blocks/image');

var _blocksImage2 = _interopRequireDefault(_blocksImage);

var _blocksHbox = require('./blocks/hbox');

var _blocksHbox2 = _interopRequireDefault(_blocksHbox);

var _blocksHero = require('./blocks/hero');

var _blocksHero2 = _interopRequireDefault(_blocksHero);

var _blocksIcon = require('./blocks/icon');

var _blocksIcon2 = _interopRequireDefault(_blocksIcon);

var _blocksContainer = require('./blocks/container');

var _blocksContainer2 = _interopRequireDefault(_blocksContainer);

var _blocksColumn = require('./blocks/column');

var _blocksColumn2 = _interopRequireDefault(_blocksColumn);

var registeredBlocks = {
	header: _blocksHeader2['default'],
	paragraph: _blocksParagraph2['default'],
	image: _blocksImage2['default'],
	hbox: _blocksHbox2['default'],
	hero: _blocksHero2['default'],
	icon: _blocksIcon2['default'],
	container: _blocksContainer2['default'],
	column: _blocksColumn2['default']
};
exports.registeredBlocks = registeredBlocks;
